gtsam/discrete/DiscreteFactorGraph.cpp

```diff
@@ -121,15 +121,25 @@ namespace gtsam {
   static DecisionTreeFactor ProductAndNormalize(
       const DiscreteFactorGraph& factors) {
     // PRODUCT: multiply all factors
-    gttic(product);
+#if GTSAM_HYBRID_TIMING
```

Comment:
That's a bit weird. So if timing is on, but hybrid is off, we will get all timing *but* hybrid? gttic and gttoc are designed to be unobtrusive and resolve to empty if the timing flag is off.